tools/install/terrascan.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -eo pipefail
+# shellcheck disable=SC1091 # Created by Dockerfile above script call
+source /.env
+
+if [[ $TERRASCAN_VERSION != false ]]; then
+  if [[ $TARGETARCH != amd64 ]]; then
+    ARCH="$TARGETARCH"
+  else
+    ARCH="x86_64"
+  fi
+  # Convert the first letter to Uppercase
+  OS="$(
+    echo "${TARGETOS}" | cut -c1 | tr '[:lower:]' '[:upper:]' | xargs echo -n
+    echo "${TARGETOS}" | cut -c2-
+  )"
```

Comment:
Since you've re-written it in Bash, we can employ Bashisms:
```suggestion
  OS="${TARGETOS^}"
```